draft-vvv-webtransport-http3.md

```diff
@@ -163,70 +163,55 @@ following mechanisms:
 An WebTransport session is terminated when the CONNECT stream that created it
 is closed.
 
-# Session IDs  {#session-ids}
-
-In order to allow multiple WebTransport sessions to occur within the same
-HTTP/3 connection, WebTransport assigns every session a unique ID, further
-referred to as session ID.  A session ID is a 62-bit number that is unique
-within the scope of an HTTP/3 connection, and is never reused even after the
-session is closed.  The client unilaterally picks the session ID.  As the IDs
-are encoded using QUIC variable length integers, the client SHOULD start with
-zero and then sequentially increment the IDs.  A session ID is considered to be
-used, and thus ineligible for new transports, as soon as the client sends a
-request proposing it.  These reuse requirements guarantee that both HTTP/3
-endpoints have a consistent view of the session ID space.
-
-The Session ID is a hop-by-hop property: if WebTransport is proxied, the same
-session can have different IDs from the client's and server's perspective.
-Because of that, session IDs SHOULD NOT be exposed to the application.
-
 # Session Establishment
 
 ## Establishing a Transport-Capable HTTP/3 Connection
 
-In order to indicate support for WebTransport, both the client and server MUST
-send an empty `http3_transport_support` transport parameter.  Endpoints MUST NOT
-use any WebTransport-related functionality unless the parameter has been
-negotiated.  The negotiation is done through a QUIC transport parameter instead
-of an HTTP/3-level setting as it allows the server to customize the transport
-parameters it intends to send based on whether the client has indicated support
-for WebTransport.
+In order to indicate support for WebTransport, both the client and the server
+MUST send a SETTINGS_ENABLE_WEBTRANSPORT value set to "1" in their SETTINGS
+frame.  Endpoints MUST NOT use any WebTransport-related functionality unless
+the parameter has been negotiated.
+
+If SETTINGS_ENABLE_WEBTRANSPORT is negotiated, support for the QUIC DATAGRAM
+extension MUST be negotiated as described in
+{{!HTTP3-DATAGRAM=I-D.schinazi-quic-h3-datagram}}; negotiating WebTransport
```

Comment:
The text mentions QUIC DATAGRAMs but this reference is h3 DATAGRAMS - I think the distinction is important